client/state/data-layer/wpcom/nps-survey/index.js

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	NPS_SURVEY_CHECK_ELIGIBILITY,
+} from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { setNpsSurveyEligibility } from 'state/nps-survey/actions';
+
+export const fetchSurveyEligibility = action => ( dispatch, getState ) => {
+
+	dispatch(
+		http(
+			{
+				method: 'GET',
+				path: `/nps/`,
+				apiVersion: '1.2',
+				} ),
+			}
+		)
+	);
+};
+
+// @todo this needs to get "data" returned somehow from the API call
+export const handleSuccess = ( action, {} ) => {
+	debug( '...Eligibility returned from endpoint.', data );
```

Comment:
note that since this is now all connected via plain data structures we are free to leave out the debug statement. we can get this information from Redux watching the actions fly by. it's in the `action.meta.dataLayer.data` property